Dockerfile

```diff
@@ -1,24 +1,21 @@
-ARG ANOL_COMMIT_HASH=08b57ecd563a1add5b4bdacdd026ffe4df603d20
+ARG ANOL_COMMIT_HASH=63de4627b24c5b886352858c2e52f60dcc3f16c0
 
-FROM node:14.19.3-alpine3.14 AS clientbuilder
+FROM node:22-alpine@sha256:6e80991f69cc7722c561e5d14d5e72ab47c0d6b6cfb3ae50fb9cf9a7b30fdf97 AS clientbuilder
 
 ARG ANOL_COMMIT_HASH
 RUN apk add --no-cache wget unzip
 
-RUN npm i -g npm@7
-
 RUN mkdir -p tmp/anol
 RUN cd /tmp/anol \
     && wget https://github.com/terrestris/anol/archive/$ANOL_COMMIT_HASH.zip -O anol.zip \
     && unzip anol.zip \
     && mv anol-$ANOL_COMMIT_HASH /anol
 
-RUN cd /anol && npm ci
+RUN cd /anol && npm ci --omit=dev
```

Comment:
Don't we need the dev dependencies for building the frontend?

Reply:
No, the anol dev dependencies are not needed. Anol is included via source files and is built by webpack in the bielfeldGEOCLIENT.